src/prefect/server/api/concurrency_limits_v2.py

```diff
@@ -17,6 +18,16 @@
 from prefect.server.utilities.schemas import PrefectBaseModel
 from prefect.server.utilities.server import PrefectRouter
 
+
+class ConcurrencyLeaseHolder(BaseModel):
+    """Model for validating concurrency lease holder information."""
+
+    model_config: ClassVar[ConfigDict] = ConfigDict(extra="forbid")
+
+    type: Literal["flow_run", "task_run", "deployment"]
+    id: str
```

Comment:
We might want this to be a UUID for added safety.
```suggestion
    id: UUID
```